argoverse/utils/interpolate.py

```diff
@@ -190,7 +189,6 @@ def interp_arc(t: int, px: np.ndarray, py: np.ndarray) -> np.ndarray:
     assert px.size == py.size
 
     pxy = np.array((px, py)).T
-    ndim = 2
```

Comment:
maybe we can add to the docstring that this is a 2-dimensional polyline

Reply:
👍